docs/mindsdb-docs/docs/connect/mysql-client.md

```diff
@@ -19,7 +19,7 @@ You can either connect locally or to a MindsDB Cloud instance, depending on the
 === "MindsDB Cloud"
 
     ```bash
-    mysql -h cloud.mindsdb.com --port 3307 -u [mindsdb_cloud_email] -p [mindsdb_cloud_password]
+    mysql -h cloud.mindsdb.com --port 3306 -u [mindsdb_cloud_email] -p [mindsdb_cloud_password]
```

Comment:
Refer to [this comment](https://github.com/mindsdb/mindsdb/pull/2745/files#r950804920)